airbyte-cdk/python/airbyte_cdk/sources/configurable/requesters/requester.py

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Any, Mapping, MutableMapping, Optional, Union
+
+import requests
+from airbyte_cdk.sources.streams.http.auth import HttpAuthenticator
```

Comment:
FYI this authenticator class has been deprecated in favor of `requests.auth.AuthBase`

Reply:
updated